content/en/platform/corda/4.10/enterprise/_index.md

```diff
@@ -4,17 +4,35 @@ menu:
   versions:
     weight: -648
   corda-enterprise-4-10:
+    identifier: about-corda-landing-4-10-enterprise
     weight: -30
-    name: Corda Enterprise Edition 4.10
+    name: Corda Enterprise 4.10
 project: corda
 section_menu: corda-enterprise-4-10
-title: Corda Enterprise Edition 4.10
+title: Corda Enterprise 4.10
 version: 'Enterprise 4.10'
 ---
 
-# Welcome to Corda Enterprise Edition 4.10
+# Corda Enterprise 4.10
```

Comment:
Links on this page seem to be broken in the preview.

Reply:
Fixed the links.